integration-tests/utils.py

```diff
@@ -19,3 +20,26 @@ def loop_until(predicate, poll_sec=5, timeout_sec=120):
         time.sleep(poll_sec)
         ok = predicate()
     return ok
+
+
+def subman_identity(subman):
+    subman_response = subman.run("identity")
+    """
+    (env) [root@kvm-08-guest21 integration-tests]# subscription-manager identity
+
+    system identity: 5c00d2c6-5bea-4b6d-8662-8680e38f0dab
+    name: kvm-08-guest21.lab.eng.rdu2.dc.redhat.com
+    org name: Donald Duck
+    org ID: donaldduck
+    environment name: env-name-01
+    """
```

Comment:
The doc string has to be placed immediately after the method name. There cannot be anything else. The doc string should explain what the method does.

Reply:
added docstring with a short description a method does.
changed a way to comment the region above